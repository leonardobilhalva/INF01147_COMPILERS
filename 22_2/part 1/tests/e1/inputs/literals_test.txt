#123
'a'
"hello world"
"escaped quote: \" "
"new line: \n"
"empty string: "