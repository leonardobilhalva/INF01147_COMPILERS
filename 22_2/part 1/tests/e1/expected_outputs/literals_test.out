Found LIT_INT
Found LIT_CHAR
Found LIT_STRING
Found LIT_STRING
Found LIT_STRING
Found LIT_STRING
Símbolo: "empty string: ", Tipo: 286
Símbolo: "escaped quote: \" ", Tipo: 286
Símbolo: "hello world", Tipo: 286
Símbolo: "new line: \n", Tipo: 286
Símbolo: #123, Tipo: 281
Símbolo: 'a', Tipo: 282
Main done! File has 6 lines